charts/stable/plex/docs/migration_guide.md

```diff
@@ -0,0 +1,251 @@
+
+Make sure truetool is installed and active (refer to the link above for the how to guide) before running the following commands:
+
+```console
+truetool --mount
+```
+
+```console
+1
+```
+
+You will get list similar like this for example.
+
+![truetool-mount-list](./img/truetool-mount-list.png)
+
+To mount the directory just enter the correct number...so for me I have to enter `51`. Verify the number you are entering is for plex-config.
+
+you will get a unmount command similar to this:
+
+```console
+zfs set mountpoint=legacy root/ix-applications/releases/plex-config/volumes/pvc-af2b9242-ecf6-4659-ace4-d601211cf448 && rmdir /mnt/truetool/plex-config
+```
+
+The old plex app is mounted in a dir for example `/mnt/truetool/plex-config`.
+
+The above assumes your app is named `plex`, if its not the dir `name` will be **different**.
+
+```console
+cd /mnt/truetool/plex-config
+```
+
+run this command to verify you see a single dir called **Library**.
+
+```console
+ls
+```
+
+Create a temp dataset in **storage** -> **datasets** for your old app plex config with the follow perms as **apps** since its our default user and group for most of our charts:
+
+![perms](./img/media-dataset-perms.png)
+
+Replace this command's path with your **OWN** `pool` and `dataset` name.
+
+```console
+rsync -rav Library /mnt/POOL/DATASET
+```
+
+Check the dir if it contains a dir called `Library` if it does then unmount the PVC.
+
+```console
+truetool --mount
+```
+
+```console
+2
+```
+
+You can delete the original app at this point.
+
+Please move on to [Migrating App Data to the New App](#migrating-app-data-to-the-new-app).
+
+## Migrating Data from HostPath
+
+If your plex config is available on your own dataset, validate its perms.
+
+![perms](./img/media-dataset-perms.png)
+
+Follow the prerequisites and then uninstall the original app. If the perms are good you can move on to [migrating to the new app](#migrating-app-data-to-the-new-app).
+
+### Migrating Data from HostPath-ix-applications
+
+This section is only for the official app where users didn't specify their own hostpath.
+
+![bad-hostpath-plex](./img/plex-config-volume-hostpath.png)
+
+::: warn
+
+Be very cautious while in this particular dataset. Run the commands carefully.
+
+:::
+
+To quickly get the correct directory run the following commands:
+
+```console
+cd /mnt/POOL/ix-applications
+```
+
+```console
+find /mnt/POOL/ix-applications -name Preferences.xml
+```
+
+You should see an output like so -> `/mnt/POOL/ix-applications/releases/plex/volumes/ix_volumes/ix-plex_config/Library/Application Support/Plex Media Server/Preferences.xml`
+
+We are only interested for this location as its the start of the config dir for plex -> `/mnt/POOL/ix-applications/releases/plex/volumes/ix_volumes/ix-plex_config/Library`.
+
+Create a temp dataset in **storage** -> **datasets** for your old app plex config with the follow perms as **apps** since its our default user and group for most of our charts:
```

Comment:
```suggestion
Create a temp dataset in **Storage** -> **Datasets** for your old app plex config with the follow perms as **apps** since its our default user and group for most of our charts:
```